business_objects/agreement.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
+from typing import Optional
+from submodules.model.session import session
+from submodules.model.business_objects import general
+from submodules.model.models import Agreement
+
+
+def get(project_id: str, item_id: str) -> Agreement:
+    return (
+        session.query(Agreement)
+        .filter(Agreement.project_id == project_id, Agreement.id == item_id)
+        .first()
+    )
+
+def get_by_xfkey(project_id: str, xfkey: str) -> Agreement:
```

Comment:
i think we should include a type here as well. uuid should still be unique but especially with the index on type (which is atm always embedding but might not be in the future)

Reply:
Will include type